src/tests/pokete/util/semantic_version_test.py

```diff
@@ -0,0 +1,111 @@
+import unittest
```

Comment:
Since you moved this from semantic_version.py maybe the old file should be removed.